readme.md

```diff
@@ -83,6 +83,7 @@
 - [Vladimir Agafonkin](https://github.com/mourner/ama) - Engineer at [Mapbox](https://www.mapbox.com/), creator of [Leaflet](http://leafletjs.com/) and a bunch of [other open source libraries](https://github.com/mourner/projects). Building the future of interactive maps.
 - [Claudio Procida](https://github.com/claudiopro/ama) - Full stack web developer, mentor, evangelist and speaker. Passionate about painting and photography.
 - [Jordan Scales](https://github.com/jdan/ama) - Engineer at [Khan Academy](https://khanacademy.org), creator of the [tota11y](http://khan.github.io/tota11y) accessibility toolkit and [isomer](http://jdan.github.io/isomer).
+- [Zeno Rocha](https://github.com/zenorocha/ama) - Developer Advocate at [Liferay](http://www.liferay.com/)
```

Comment:
Needs the period. https://github.com/sindresorhus/amas/blob/master/contributing.md

```diff
- [Liferay](http://www.liferay.com/)
+ [Liferay](http://www.liferay.com/).
```